examples/quantizing_moe/deepseek_r1_example.py

```diff
@@ -20,7 +20,7 @@
     model_id, torch_dtype="auto", config=config
 )
 tokenizer = AutoTokenizer.from_pretrained(model_id)
-model = prepare_for_calibration(model)
+model = replace_modules_for_calibration(model)
```

Comment:
i think this is a better name change, but might hit merge conflicts with the changes in #1608

Reply:
yes will resolve